pkg/authorization/authorizationutil/subject.go

```diff
@@ -0,0 +1,56 @@
+package authorizationutil
```

Comment:
wow, authorizationutil. Why this name?

Reply:
i would just call it pkg/authorization

Reply:
> wow, authorizationutil. Why this name?

auto-completes, like `http/httptest` (when they realized their advice was bad).  this particular one is often abbreviated in our codebase today.